### PR TITLE
Suspend video

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

**Comment by paweldomas** [comment]
Hmm how does it work ? If we're not doing signalling updates then will not it trigger video track muted event on the receiver's side ?

File: modules/RTC/RTCUtils.js [hunk 1]
```diff
@@ -821,16 +821,13 @@ var RTCUtils = {
                             : SDPUtil.filter_special_chars(id));
                 };
                 // DTLS should now be enabled by default but..
```
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: Maybe the comment should be removed too?

**Comment by bgrozev** [comment]
Video is stopped if the bandwidth estimation for the sender is below the minimum (something very low, I think 30kbps). It will trigger no events on the receiver.

File: modules/RTC/RTCUtils.js [hunk 2]
```diff
@@ -821,16 +821,13 @@ var RTCUtils = {
...
-                this.pc_constraints = {'optional': [
-                    {'DtlsSrtpKeyAgreement': 'true'}
```
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: Why do we need this change?

File: modules/RTC/RTCUtils.js [hunk 3]
```diff
@@ -821,16 +821,13 @@ var RTCUtils = {
...
-                ]};
+                this.pc_constraints = {'optional': [] };
+
                 if (options.useIPv6) {
                     // https://code.google.com/p/webrtc/issues/detail?id=2828
                     this.pc_constraints.optional.push({googIPv6: true});
                 }
-                if (RTCBrowserType.isAndroid()) {
```
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: Why do we need this change?

File: modules/RTC/RTCUtils.js (hunk 1 above)
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: Sorry I didn't noticed!

File: modules/RTC/RTCUtils.js (hunk 3 above)
Comment (hristoterezov) [inline modules/RTC/RTCUtils.js]: I don't see problem but I haven't tested it. I wasn't sure what's the reason behind this removal. But why we are removing "DtlsSrtpKeyAgreement"? Something changed on the latest versions?

Comment (bgrozev) [inline modules/RTC/RTCUtils.js]: Not in recent versions. I don't know when it was last required to use this constraint, but it wasn't recently.

File: modules/RTC/RTCUtils.js (hunk 2 above)
Comment (bgrozev) [inline modules/RTC/RTCUtils.js]: See below.

**Pull request merged**